src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
+	static String copyright = "Team Blue";
+  static String version = "v1.0";
+  static boolean exit_prompt = false;
+
+	static Scanner scanner = new Scanner(System.in).useDelimiter(";");
+
+  public String readStatement(String input) throws IOException {
+
+    PromptScreen();
+
+		String userInput = "";
+
+		System.out.print(prompt);
+    userInput = scanner.next().replace("\n", " ").replace("\r", "").trim().toLowerCase();
```

Comment:
There are multiple bugs in this line.

- What if a DavisBase TEXT literal value contains embedded `\n` or `\r`? This statement would incorrectly clobber them.
- What if a DavisBase TEXT literal value contains embedded `;`? This statement would incorrectly split in the middle of a TEXT value.
- What if a DavisBase TEXT literal value contains any upper case characters? This statement would incorrectly clobber them.